kortex_driver/readme.md

```diff
@@ -213,6 +214,50 @@ Many things have been changed in the ros_kortex repository between versions 1.1.
 * The **/my_robot_name/base_feedback/joint_state** topic is now advertised as **/my_robot_name/joint_state**.
 * The [kortex_driver launch file](launch/kortex_driver.launch) is now located in the `kortex_driver` package instead of the `kortex_bringup` package, which was deleted. Some arguments were added to the file.
 
+
+<a id="multiple"></a>
+## Support for multiple arms
```

Comment:
maybe add a line somewhere stating that this is not an example and it is more designed to show people how to launch two arms, that they will most likely have to modify this to fit their own integration of our arms in their setup